htsinfer/infer_read_orientation.py

```diff
@@ -22,5 +21,45 @@ def infer(
         LIBTYPE string according to Salmon documentation, cf.
         https://salmon.readthedocs.io/en/latest/library_type.html
     """
+    #create .tmp directory in "inputDir" and retuns .tmp path
+    tmpDir = mkTmpDir(inputDir)
+    
+    # call functions of issues #25, 26, 27 and 34
+    
+    # delete .tmp
+    rmTmpDir(tmpDir)
+        
     # implement logic
     return "U"
+    
+
+def mkTmpDir(inputDir):
+    """Function to create a temporary directory for the storage of 
+    temporary data."""
+  
+    import os
+
+    # define the name of the directory to be created
+    tmpPath = inputDir + "/.tmp"
+  
+    try:
+        os.mkdir(tmpPath)
+    except OSError:
+      print ("Creation of the directory %s failed" % tmpPath)
+    else:
+      print ("Successfully created the directory %s " % tmpPath)
+  
+    return (tmpPath)
+      
+      
+def rmTmpDir(tmpPath):
+    """Function to remove the temporary directory."""
+  
+    import os
+    #remove temporary directory
+    try:
+        os.rmdir(tmpPath)
```

Comment:
This only deletes an _empty_ directory. Instead you need to remove the directory and all its contents. See here for how to do this: https://stackoverflow.com/questions/6996603/how-to-delete-a-file-or-folder

Again, you won't need a function for that, just put it at the end of the `infer()` function, just before the `return` statement. Add the `try - except` and log messages as described above.